packages/webapp/src/containers/Animals/Inventory/KPI/index.tsx

```diff
@@ -115,7 +115,7 @@ function KPI({ isCompactSideMenu, selectedTypeIds, onTypeClick }: KPIProps) {
         <PureTileDashboard
           typeCountTiles={types}
           dashboardTitle={t('SECTION_HEADER.ANIMALS_INVENTORY')}
-          categoryLabel={t('TYPES')}
+          categoryLabel={t('common:TYPES')}
```

Comment:
Passing the filename to `useTranslation` works in-app, but the `pnpm i18n` script did not detect it and generated new keys within `translation.json`